### PR TITLE
Improve 55 Add error messages to GooglePlay verification

### PR DESCRIPTION
**Pull request by EugeneButusov**

### CODE REVIEW

File: src/com/soomla/store/billing/google/SoomlaGpVerification.java [hunk 1]
```diff
@@ -124,6 +124,11 @@ public void verifyPurchase() {
                             } else {
                                 errorCode = UnexpectedStoreErrorEvent.ErrorCode.VERIFICATION_FAIL;
                                 SoomlaUtils.LogError(TAG, "Failed to verify transaction receipt. The user will not get what he just bought.");
+                                try {
+                                    SoomlaUtils.LogError(TAG, "Reason: " + resultJsonObject.getJSONObject("reason").getJSONObject("error").getString("message"));
```
Comment (vedi) [inline src/com/soomla/store/billing/google/SoomlaGpVerification.java:128]: @EugeneButusov, please, make this call safer, checking "reason", "error" for null.

Comment (EugeneButusov) [inline src/com/soomla/store/billing/google/SoomlaGpVerification.java:128]: @vedi, I thought between this and exception handling and supposed that try-catch block is more elegant than `if` condition with 3 boolean expressions.

Comment (vedi) [inline src/com/soomla/store/billing/google/SoomlaGpVerification.java:128]: @EugeneButusov, it is, if it handles all the cases.

Comment (vedi) [inline src/com/soomla/store/billing/google/SoomlaGpVerification.java:128]: @EugeneButusov, looks like it's ok in your case. Sorry. Scratch it out.

**Pull request merged**